aws_lambda_powertools/utilities/feature_flags/feature_flags.py

```diff
@@ -228,7 +244,7 @@ def evaluate(self, *, name: str, context: Optional[Dict[str, Any]] = None, defau
         # method `get_matching_features` returning Dict[feature_name, feature_value]
         boolean_feature = feature.get(
             schema.FEATURE_DEFAULT_VAL_TYPE_KEY, True
-        )  # backwards compatability ,assume feature flag
+        )  # backwards compatibility ,assume feature flag
```

Comment:
while we are here :) 

```suggestion
        )  # backwards compatibility, assume feature flag
```

Reply:
fixed